### PR TITLE
Merge MI-108 with dev 

### PR DESCRIPTION
**Pull request by AlineTheAlien**
Changed setGSD method to only accept a dynamic object from body

### CODE REVIEW

**Comment by AlineTheAlien** [comment]
@nim442 Can you please review?

File: src/mRides-server/Controllers/UserController.cs [hunk 1]
```diff
@@ -98,10 +98,11 @@ public long getGSD(int id)
         /// </summary>
         // POST api/values
         [HttpPost]
-        public long setGSD([FromHeader]string id, [FromBody]dynamic sentObject)
+        public long setGSD([FromBody]dynamic sentObject)
         {
-            int newUserId = Convert.ToInt32(id);
+            //int newUserId = Convert.ToInt32(userId);
```
Comment (nim442) [inline src/mRides-server/Controllers/UserController.cs]: remove commented out code

File: src/mRides-server/Logic/UserCatalog.cs [hunk 2]
```diff
@@ -144,15 +144,13 @@ public User create(User obj, int userId)
 
         public void setGSD(int userId, long amountGSD)
         {
-            //User u = _context.Users.Find(userId);
```
Comment (nim442) [inline src/mRides-server/Logic/UserCatalog.cs:147]: 👍

**Review by nim442 — CHANGES_REQUESTED** [review]

File: src/mRides-server/Controllers/UserController.cs (hunk 1 above)
Comment (AlineTheAlien) [reply src/mRides-server/Controllers/UserController.cs]: Good catch!

**Comment by AlineTheAlien** [comment]
@nim442 I made the changes you requested, the branch is ready to merge.

**Pull request merged**